### PR TITLE
Show PR test status and phase on PR Dashboard.

### PR DESCRIPTION
**Pull request by rmmh**

### CODE REVIEW

**Comment by spxtr** [comment]
Picture?

File: gubernator/github/classifier.py [hunk 1]
```diff
@@ -37,7 +37,18 @@ def classify_issue(repo, number):
     events.sort(key=lambda e: e.timestamp)
     logging.debug('classifying %s %s (%d events)', repo, number, len(events))
     event_pairs = [(event.event, json.loads(event.body)) for event in events]
-    return tuple(list(classify(event_pairs)) + [events[-1].timestamp])
+
+    last_event_timestamp = events[-1].timestamp
```
Comment (spxtr) [inline gubernator/github/classifier.py]: Is it possible for events to be empty here?

Comment (rmmh) [inline gubernator/github/classifier.py]: No, since we only run classify_issue when we receive an event for that issue.

**Comment by spxtr** [comment]
That's pretty awesome.

**Pull request merged**